src/git/fixture_test.go

```diff
@@ -35,6 +36,48 @@ func setup() *fixture {
 	return &fixture{log: logger, gitLabVersioning: &GitLabVersioningMock{}}
 }
 
+func getValidSetup() *fixture {
+	f := setup()
+	f.gitLabVersioning.url = "https://gitlab/dataplatform/integration-tests.git"
+	f.gitLabVersioning.destinationDirectory = fmt.Sprintf("%s/%s", os.Getenv("HOME"), "integration-tests")
+	f.gitLabVersioning.username = "root"
+	f.gitLabVersioning.password = "password"
+	return f
+}
+
+func newValidVersion(currentVersion string) (string, error) {
+	splitedVersion := strings.Split(currentVersion, ".")
+	fmt.Println(splitedVersion)
```

Comment:
should we use log instead fmt? @esequielvirtuoso

Reply:
Oh, that should not be there.
It was a test. :D, I'll remove it.